x-pack/plugins/security_solution/public/assistant/content/conversations/index.tsx

```diff
@@ -70,35 +65,7 @@ export const BASE_SECURITY_CONVERSATIONS: Record<string, Conversation> = {
       },
       user: {},
     },
-    messages: [
-      {
-        role: 'assistant',
-        content: WELCOME_GENERAL,
-        timestamp: '',
-        presentation: {
-          delay: 2 * 1000,
-          stream: true,
-        },
-      },
-      {
-        role: 'assistant',
-        content: WELCOME_GENERAL_2,
-        timestamp: '',
-        presentation: {
-          delay: 1000,
-          stream: true,
-        },
-      },
-      {
-        role: 'assistant',
-        content: WELCOME_GENERAL_3,
-        timestamp: '',
-        presentation: {
-          delay: 1000,
-          stream: true,
-        },
-      },
-    ],
+    messages: [],
```

Comment:
just noting that with this change, the welcome messages are no longer initialized here